src/web/middleware/jira/jira-admin-only-auth-middleware.ts

```diff
@@ -27,5 +30,14 @@ export const jiraAdminOnlyAuthMiddleware = (
 
 			next();
 		})
-		.catch(next);
+		.catch((err) => {
+			if (err instanceof ForbiddenByJiraServiceError) {
+				return next(
+					new UnauthorizedResponseStatusError(
```

Comment:
For this bit, we're mapping an underlying 403 error to a 401 error. I don't think we need to expose the underlying 403 error, so this seems alright to me.